src/MessageHandler/ExportOrdersHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AppBundle\MessageHandler;
+
+use AppBundle\Message\ExportOrders;
+use AppBundle\Sylius\Taxation\TaxesHelper;
+use AppBundle\Utils\RestaurantStats;
+use Doctrine\ORM\EntityManagerInterface;
+use Knp\Component\Pager\PaginatorInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ExportOrdersHandler implements MessageHandlerInterface
+{
+
+    public function __construct(
+        private TranslatorInterface $translator,
+        private EntityManagerInterface $entityManager,
+        private PaginatorInterface $paginator,
+        private TaxesHelper $taxesHelper,
+        private ContainerInterface $container
+    )
+    { }
+
+    public function __invoke(ExportOrders $message): ?string
+    {
+        $stats = new RestaurantStats(
+            $this->entityManager,
+            $message->getFrom()->setTime(0, 0, 1),
+            $message->getTo()->setTime(23, 59, 59),
+            null,
+            $this->paginator,
+            $this->container->getParameter('kernel.default_locale'),
```

Comment:
We should be able to override the locale via a property in the `ExportOrders` message. 
Because for the exports we are interested in, it is easier to have a stable language, i.e English.